decidim-core/app/services/decidim/action_authorizer.rb

```diff
@@ -47,6 +54,8 @@ def status_data
         [:invalid, fields: unmatched_fields]
       elsif missing_fields.any?
         [:incomplete, fields: missing_fields]
+      elsif authorization_expired?
```

Comment:
Shouldn't `authoriation_expired?` take precedence over all the previous ones? Otherwise, you could get an `:incomplete` dialog when you actually need to fulfill the whole authentication. Not a big issue, but still.

Reply:
Makes sense, yes.